components/automate-ui/src/app/components/process-progress-bar/process-progress-bar.component.scss

```
@@ -17,15 +16,23 @@
   }
 
   button {
+    margin: 8px;
+    border: 1px solid $chef-white;
     position: absolute;
     right: 0;
     top: 0;
   }
 
+  button:hover,
+  button:focus {
+    background-color: #174AF0;
+  }
+
   mat-progress-bar {
     border-radius: 5px;
     height: 6px;
     margin-top: 5px;
+    width: calc(100% - 175px);
 
     .mat-progress-bar-fill::after {
       background-color: rgba(255, 255, 255, .7);
```

I realize this isn't your work here, but if you end up back in this file and feel like doing a little cleanup I'd pass in the variable. 
```suggestion
      background-color: rgba($chef-white, .7);
```

---

same thing down on line 42 also